### PR TITLE
Create Release Reminders

### PR DESCRIPTION
**Pull request by ridz1208**
Historically we keep forgetting to do at least 1 task at every release. Instead of working on improving our memory I suggest actually checking boxes.

@driusan note how I did not use the word checklist in the title

### CODE REVIEW

File: release_checklist.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- [ ] Create SQL Release patch into SQL release directory and move cleanup SQL patches
+- [ ] composer update ?
+- [ ] Update README and install instructions dependencies
+- [ ] Create release notes and upgrade instructions
+- [ ]
```
Comment (cmadjar) [inline release_checklist.md:5]: no idea if those should be added to the reminders? Just dropping as I think about them
```suggestion
- [ ] is there any VERSION documentation that needs to be updated?
- [ ] tag all PRs going into the release with proper milestone and review the ones that have "add to release notes"
- [ ] for major and minor releases, create new branches out of the release for the bug fix to go into
- [ ] pull release into main and other appropriate branches if it is bug fix release
```

File: release_checklist.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+- [ ] Create SQL Release patch into SQL release directory and move cleanup SQL patches
+- [ ] composer update ?
```
Comment (driusan) [inline release_checklist.md:2]: shouldn't this be done before release testing?

Comment (ridz1208) [reply release_checklist.md:2]: sure we can reorganize this list

**Comment by driusan** [comment]
The description for Projects (classics) says "Projects (classic) 
Projects (classic) are the legacy version of Projects and are created at the repository level. You can create projects for single repository feature work, simple sprint planning, or **release checklists.**"

Maybe this would be good to turn into a project instead of a PR?

**Pull request closed without merging**